notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb

```diff
@@ -474,7 +428,7 @@
         "id": "rough-alert"
       },
       "source": [
-        "# Run AutoML Video Training with Vertex AI Video Dataset"
+        "## Run AutoML Video Training with Vertex AI Video Dataset"
```

Comment:
Change "Run AutoML Video Training with Vertex AI Video Dataset" to:

- Run AutoML video training with Vertex AI video dataset